plugins/neo4j-adapter/src/main/java/org/polypheny/db/adapter/neo4j/util/Translator.java

```diff
@@ -191,11 +191,11 @@ private String handleBinaries( RexCall call ) {
         RexNode rightRex = call.operands.get( 1 );
         String left = leftRex.accept( this );
         if ( leftRex.isA( Kind.LITERAL ) && PolyType.STRING_TYPES.contains( leftRex.getType().getPolyType() ) ) {
-            left = String.format( "'%s'", left );
+            //left = String.format( "'%s'", left );
         }
         String right = rightRex.accept( this );
         if ( rightRex.isA( Kind.LITERAL ) && PolyType.STRING_TYPES.contains( rightRex.getType().getPolyType() ) ) {
-            right = String.format( "'%s'", right );
+            //right = String.format( "'%s'", right );
```

Comment:
This looks like an artifact from testing something.

Reply:
Removed unnecessary if statements.